bpo-36896: Clarify that some types constructors are unstable (#13271)
python/cpython: Doc/library/types.rst

Typical use of these names is for :func:`isinstance` or
:func:`issubclass` checks.

> .. note::

Reviewer: Delete this and just add a revised sentence (next comment) to the previous paragraph.
```suggestion

```

>    These types are not supposed to be instantiated outside of CPython internals and constructor signatures will vary between python versions.

Reviewer 2: `python` should be capitalized.

Author: Oops indeed, good catch thanks, fixed.

>    These types are not supposed to be instantiated outside of CPython internals and constructor signatures will vary between Python versions.

Reviewer: The 'not supposed to be' part may be true for some of the types not listed, but is definitely not true for at least some of the types.
```suggestion
   If you instantiate any of these types, note that signatures may vary between Python versions.
```
